src/filecoin/filecoinInfo.ts

```diff
@@ -1,3 +1,4 @@
+import { NetworkPrefix } from '@zondax/izari-filecoin'
```

Comment:
We cannot currency libraries from the info files, either directly or transitively. However, `import type` statements are safe to use, because those get compiled out.

So, put this definition in `filecoinTypes.ts`:

```ts
import type { Network, NetworkPrefix } from '@zondax/izari-filecoin'

export interface FilecoinNetworkInfo {
   ...
  networkPrefix: keyof typeof NetworkPrefix
}
```

Now this will be equivalent to `networkPrefix: 'Mainnet' | 'Testnet'`, which means `filecoinInfo.ts` can just do `networkPrefix: 'Mainnet'` with no run-time imports.

Finally, when you go to use this flag in `Wallet.deriveAccount`, you will have to do `import { NetworkPrefix } from '@zondax/izari-filecoin'`, and then you can do `NetworkPrefix[this.networkInfo.networkPrefix]` to translate the enum key into the enum value.